src/main/resources/db/changelog/changes/db.changelog-4.1.0.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+  <changeSet id="MODEXPS-37@@rename the enum type you want to change" author="d.butramyou">
+    <sql dbms="postgresql">
+      ALTER TYPE ExportType rename TO _export_type;
+    </sql>
+  </changeSet>
+
+  <changeSet id="MODEXPS-37@@create new type" author="d.butramyou">
+    <sql dbms="postgresql">
+      CREATE TYPE ExportType AS enum ('CIRCULATION_LOG', 'BURSAR_FEES_FINES',
```

Comment:
Can we do that without table recreation from scratch with old and new ExportTypes?